assets/base.css

```diff
@@ -2280,8 +2293,8 @@ product-info .loading-overlay:not(.hidden) ~ *,
 
 .header {
   display: grid;
-  grid-template-areas: 'left-icon heading icons';
-  grid-template-columns: 1fr 2fr 1fr;
+  grid-template-areas: 'left-icons heading icons';
+  grid-template-columns: 1fr auto 1fr;
```

Comment:
Good find!

Reply:
I agree about changing `auto` to `2fr` back as it used to be. I'm going to leave `left-icons` to avoid adding more logic because in some cases it comes together with hamburger icon.

Reply:
But on mobile do we ever have 2 icons ? I don't believe so. 
So there shouldn't be a need to change the area name 🤷

Reply:
We agreed to keep it as`left-icons`